extensions/ipynb/.vscodeignore

```diff
@@ -1,6 +1,6 @@
 .vscode/**
 src/**
-out/**
+notebook-src/**
```

Comment:
`out/` still should be listed here. You just also want to include `notebook-src`